lib/api/apiUtils/authorization/aclChecks.js

```diff
@@ -73,6 +74,13 @@ function isBucketAuthorized(bucket, requestType, canonicalID) {
             || bucketAcl.FULL_CONTROL.indexOf(canonicalID) > -1
             || bucketAcl.WRITE.indexOf(canonicalID) > -1) {
             return true;
+        } else if (bucketAcl.WRITE.indexOf(publicId) > -1
```

Comment:
both of the blocks return `true`, so can be merged.

Reply:
I believe this was to match the other conditionals in the method, I cherry-picked it from a commit by @naren-scality